### PR TITLE
rfc: enforcement steps in coc

### PR DESCRIPTION
**Pull request by zkat**
This RFC proposes an update to our CoC and enforcement policies to make enforcement clearer to users.

### CODE REVIEW

File: _pages/rfc-unify-coc-enforcement.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# RFC - Unify CoC and Enforcement policies
+
+## tl;dr
```
Comment (varjmes) [inline _pages/rfc-unify-coc-enforcement.md]: Can we do 'In Summary' here, instead? :)

File: _pages/rfc-unify-coc-enforcement.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+* Move the enforcement steps to the top of the CoC.
+* Keep the separate enforcement document as a detailed policy that includes lessons/values/nuance.
+* Stop directing people to both the CoC and enforcement policy: on the signup page, on docs, etc.
+* Put a link to the full enforcement values doc in the CoC.
+
+## Intro
+
+The first thing people should see when they go to the CoC is how it affects them. The bulk of the stuff on the CoC itself amounts to legalese that most users won't read -- so in the interest of conveying information that matters right away, it's best to lead with how CoC violations are handled. I would argue that's the main thing people even are about.
```
Comment (varjmes) [inline _pages/rfc-unify-coc-enforcement.md]: "the main thing people even are about." missed a "c" on "are".

File: _pages/rfc-unify-coc-enforcement.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Hopefully, these changes will make expectations even clearer for everone!
```
Comment (varjmes) [inline _pages/rfc-unify-coc-enforcement.md]: "everone" -> "everyone"

File: _pages/rfc-unify-coc-enforcement.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+## Changes
+
+### Move enforcement steps to CoC
+
+There's two sections of enforcement steps in the enforcement doc. Combine them, reword them, and put them on the very top of the main CoC.
+
+```
+If you see a CoC violation, these are the steps that should be followed:
+
+1. Tell the person “hey, that’s not ok”.
```
Comment (varjmes) [inline _pages/rfc-unify-coc-enforcement.md]: I prefer something like "Let the person know that what they're doing is not appropriate"

File: _pages/rfc-unify-coc-enforcement.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+## Admin Enforcement
+
+Once the admins get involved, they will follow a documented series of steps and do their best to preserve the well-being of WeAllJS members. This section covers actual concrete steps. For further information/details on values and practices that admins will generally apply when enforcing the Code of Conduct, refer to [the full enforcement document](http://wealljs.org/enforcement).
+
+### Contacting Admins
+
+You may get in touch with the WeAllJS admin team through any of the following methods:
+
+  * Use the `/admin` command in the WeAllJS Slack to send a message directly to the admin channel.
+  * Direct Message [@WeAllJS](https://twitter.com/wealljs) on Twitter with the details and any relevant links.
```
Comment (varjmes) [inline _pages/rfc-unify-coc-enforcement.md]: Can we order these by which we'd prefer?

I'd like:

- /admin
- DM in Slack
- Email
- DM WeAllJs

File: _pages/rfc-unify-coc-enforcement.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+In Slack, `?admins` will list the names of all current admins.
+
+### Further Enforcement
+
+If you've already followed the [initial enforcement steps](#enforcement), these are the steps admins will take for further enforcement, as needed:
+
+  1. Repeat the request to stop.
+  2. If the person doubles down, they will be removed from the channel and given an official warning.
+  3. If the behavior continues or is repeated later, the person will be deactivated for 24 hours.
+  4. If the behavior continues or is repeated after the temp deactivation, a long-term (6-12mo) deactivation will be used.
```
Comment (varjmes) [inline _pages/rfc-unify-coc-enforcement.md]: can we say "temporary" instead of "temp?". Just for language consistency 🤓

**Review by varjmes — CHANGES_REQUESTED** [review]

File: _pages/rfc-unify-coc-enforcement.md (hunk 4 above)
Comment (cjoulain) [reply _pages/rfc-unify-coc-enforcement.md]: I know it's not easy for everyone but I do like that this encourages someone noticing a CoC violation be pro-active about commenting on it. Do you think this write-up should give examples? (For example, something along the lines of: "If you don't feel comfortable addressing a person who made a problematic remark in the channel, feel free to PM them.) I think it empowers the person noticing a remark to manage the situation and the conversation might not need to be escalated to admin.

**Comment by zkat** [comment]
@Charlotteis @cjoulain updated! lmk what you think. I added a whole examples section

**Review by varjmes — APPROVED** [review]

**Comment by varjmes** [comment]
:shipit: 😻

File: _pages/enforcement.md [hunk 7]
```diff
@@ -4,18 +4,20 @@ title: "Enforcement"
...
-  2. That person should apologize immediately and correct the behavior.
-  3. If this doesn't happen, isn't safe to do, or is not satisfactory, message admins.
-  4. Admins will take appropriate action in a timely manner, prioritizing those who were hurt.
+# Code of Conduct Enforcement Philosophy
 
 ## Introduction
 
-No matter how kind or nice people think they are, at some point, someone's gonna say Some Shit:tm: and things are going to go awry. This document is meant as a writeup and guideline holding core values about what this community considers the best approach to resolving these conflicts in a kind and positive way, as much as possible. It is paired with the community's [Code of Conduct](/code-of-conduct). It is a living document, informed by new problems, new conflicts, and always strives to help make the WeAllJS community successful while including the voices of those who are too often not heard.
+No matter how kind or nice people think they are, at some point, someone's gonna
+say Some Shit:tm: and things are going to go awry. This document is meant as a
```
Comment (jenniferlynparsons) [inline _pages/enforcement.md:12]: Is the profanity okay? The document is fairly informal, but I'm not sure exactly how informal it should be.

Comment (zkat) [reply _pages/enforcement.md:12]: I don't know. To me, that's an undirected phrase that hopefully adds a hint of levity to a more serious document. I'm happy to change it though.

Comment (jenniferlynparsons) [reply _pages/enforcement.md:12]: That's pretty much how I react to it as well, and have no personal problem with it staying in.

**Pull request merged**